packages/components/descriptions/index.ts

```diff
@@ -1,7 +1,7 @@
 import { withInstall, withNoopInstall } from '@element-plus/utils/with-install'
 
-import Descriptions from './src/index.vue'
-import DescriptionsItem from './src/description-item'
+import Descriptions from './src/descriptions.vue'
+import DescriptionsItem from './src/descriptions-item'
```

Comment:
We need to expose the types like [this](https://github.com/element-plus/element-plus/blob/b6f61dd562107eab5660c6cb7662237c9cab8f7f/packages/components/button/src/button.ts#L55) does